client/actions/DocumentAction.js

```diff
@@ -0,0 +1,86 @@
+import axios from 'axios';
+import omit from 'lodash/omit';
+import jwtDecode from 'jwt-decode';
+import types from './actionTypes';
+
+/**
+//  * createDocument - create documents action
+ * @param  {object} documents document data
+ * @return {object} return an object
+ */
+export function loadDocumentSuccess(documents) {
+  return {
+    type: types.LOAD_DOCUMENTS_SUCCESS,
+    documents
+  };
+}
+
+/**
+ * loadDocument - fetches documents from database
+ * @return {Function} returns a dispatch
+ */
+export function loadDocuments() {
+  const token = localStorage.getItem('jwtToken');
+  const details = jwtDecode(token);
+  const userId = details.user.id;
+  return dispatch => axios.get(`/users/${userId}/documents`)
+    .then((response) => {
+      const documents = response.data;
+      dispatch(loadDocumentSuccess(documents.document));
+    })
+    .catch((error) => {
+      throw (error);
+    });
+}
+
+/**
+ * createDocument - creates a new document
+ * @param  {object} data contains the document data
+ * @return {Function} returns a dispatch
+ */
+export function createDocument(data) {
+  return dispatch => axios.post('/documents', data)
+    .then(() => {
+      dispatch(loadDocuments());
+    })
+    .catch((error) => {
+      throw (error);
```

Comment:
Please, do not throw errors you are not handling, ensure to dispatch a failure action